test/unit/specs/components/Personalization/createCollect.spec.js

```diff
@@ -18,7 +18,10 @@ describe("Personalization::createCollect", () => {
     id: 1,
     decisionId: "foo"
   };
-  const event = { type: "blah" };
+  const event = {
+    type: "blah",
```

Comment:
Is `type: "blah"` here serving any purpose?

Reply:
Removed.